CHANGELOG.md

```diff
@@ -63,7 +63,8 @@ of the migration as it runs, you can view the CloudWatch logs for your async
 operations (e.g. `PREFIX-AsyncOperationEcsLogs`).
 
 ### Breaking Changes
-
+- **CUMULUS-3689**
+ - updated aggregate_stats api endpoint to use postgresql instead of elasticsearch
```

Comment:
- packages/api/endpoints/stats.js aggregate function is not updated to use `new StatsSearch`.
- The queryStringParameters should be an object. 
- If the existing unit tests for stats endpoint are not enough, add more.